workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Delegates.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Improbable.Worker;
+
+namespace Improbable.Gdk.Core.MonoBehaviours
+{
+    // TODO reviewers: shall we put these delegates into their own class/namespace?
+    public delegate void AuthorityChangedDelegate(Authority newAuthority);
+
+    public delegate void ComponentUpdateDelegate<in TComponentUpdate>(TComponentUpdate updateData)
```

Comment:
Is there any reason for the contravariant parameter here?

Reply:
Only the recommendation from rider